lib/src/track/remote/video.dart

```diff
@@ -18,4 +20,82 @@ class RemoteVideoTrack extends RemoteTrack with VideoTrack {
           track,
           receiver: receiver,
         );
+
+  VideoReceiverStats? prevStats;
+  num? _currentBitrate;
+  get currentBitrate => _currentBitrate;
+
+  @internal
+  String? getDecoderImplementation() {
+    return prevStats?.decoderImplementation;
+  }
+
+  @override
+  Future<void> monitorReceiver() async {
+    if (receiver == null) {
+      _currentBitrate = 0;
+      return;
+    }
+    final stats = await getReceiverStats();
+
+    if (stats != null && prevStats != null && receiver != null) {
+      _currentBitrate = computeBitrateForReceiverStats(stats, prevStats);
+      events.emit(VideoReceiverStatsEvent(
+          stats: stats, currentBitrate: currentBitrate));
+    }
+
+    prevStats = stats;
+  }
+
+  Future<VideoReceiverStats?> getReceiverStats() async {
+    if (receiver == null) {
+      return null;
+    }
+
+    final stats = await receiver!.getStats();
+    VideoReceiverStats? receiverStats;
+    for (var v in stats) {
+      if (v.type == 'inbound-rtp') {
+        receiverStats ??= VideoReceiverStats();
+        receiverStats.timestamp = v.timestamp;
+        receiverStats.streamId = v.id;
+        receiverStats.jitter ??= getNumValFromReport(v.values, 'jitter');
```

Comment:
just out of curiosity: what's the reason why some of the properties are getting assigned directly and some with a null aware assignment?

Reply:
This is because `id` and `timestamp` are in the fixed parameters of StatsReport, and `jitter/packetReceived` are in the kv map, so the parameters are always uncertain.